packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { boolean, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import {
+  getDoublePica,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import { C_CLOUD_LIGHT, C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+
+const TOP_MARGIN_PX = 1;
+
+const calcTop = ({ inline, script }) =>
```

Comment:
I know eslint is making it this way, but is there anyway to appease it, while making it look nice and easy to read? :P

Reply:
How does 722d3ab look?